sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ * Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Github is seeing this as a file move but I actually manually created this new file.

Reply:
git looks for diffs and tries to guess what was a move and what was not.